tensorflow/python/keras/datasets/mnist.py

```diff
@@ -47,8 +47,8 @@ def load_data(path='mnist.npz'):
       path,
       origin=origin_folder + 'mnist.npz',
       file_hash='8a61469f7ea1b51cbae51d4f78837e45')
-  f = np.load(path)
-  x_train, y_train = f['x_train'], f['y_train']
-  x_test, y_test = f['x_test'], f['y_test']
-  f.close()
+  with np.load(path) as f:
+    x_train, y_train = f['x_train'], f['y_train']
+    x_test, y_test = f['x_test'], f['y_test']
+    
   return (x_train, y_train), (x_test, y_test)
```

Comment:
You can include the return statement in the with context as well.

Reply:
Fixed. I have also updated `boston_housing` file.